doc/faq.rst

```diff
@@ -385,3 +385,7 @@ efficient to process for most operations. Extensive work would also be needed
 to support Pandas categorical types. Restricting input to homogeneous
 types therefore reduces maintenance cost and encourages usage of efficient
 data structures.
+
+Do you plan to implement transform for target y in a pipeline?
+----------------------------------------------------------------------------
+Currently transform only works for features X in a pipeline. There's a long-standing discussion about not being able to transform y in a pipeline. Follow on github issue #4143. Meanwhile check out sklearn.compose.TransformedTargetRegressor, `pipegraph<https://github.com/mcasl/PipeGraph>`_, `imbalanced-learn<https://github.com/scikit-learn-contrib/imbalanced-learn>`_
```

Comment:
We like to keep every line under 80 characters, so you have to split this long line into multiple lines

```suggestion
Currently transform only works for features X in a pipeline. There's a
long-standing discussion about not being able to transform y in a pipeline.
Follow on github issue #4143. Meanwhile check out
sklearn.compose.TransformedTargetRegressor,
`pipegraph<https://github.com/mcasl/PipeGraph>`_,
`imbalanced-learn<https://github.com/scikit-learn-contrib/imbalanced-learn>`_
```

Reply:
Other comments:
- Can you add a link on `#4143` to be able to go directly to this discussion?
- You can change `sklearn.compose.TransformedTargetRegressor` into
```py
:class:`sklearn.compose.TransformedTargetRegressor`
```
which will automatically link to the corresponding page in the documentation.

Reply:
@TomDLT Changes made.